app/cli/cmd/attestation_push.go

```diff
@@ -19,10 +19,11 @@ import (
 	"errors"
 	"fmt"
 
-	"github.com/chainloop-dev/chainloop/app/cli/internal/action"
 	"github.com/spf13/cobra"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+
+	"github.com/chainloop-dev/chainloop/app/cli/internal/action"
```

Comment:
These changes make the PR very noisy, you mentioned that you use `goimports` but I can not reproduce the change locally with latest version of goimports.

Are you sure you do not have some additional configuration in the editor?

```
# From main
goimports -w app/cli/cmd/attestation_push.go 
git status
[empty]
```

I think we need to figure this out before we merge, otherwise we'll be doing back and forth continuously

Reply:
Ok, let me undo these accidental changes, since they are not related.